[2025-09-24T09:24:15.894+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sales_dag.create_schemas_tables scheduled__2025-09-23T00:00:00+00:00 [queued]>
[2025-09-24T09:24:15.918+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sales_dag.create_schemas_tables scheduled__2025-09-23T00:00:00+00:00 [queued]>
[2025-09-24T09:24:15.920+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-09-24T09:24:15.993+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): create_schemas_tables> on 2025-09-23 00:00:00+00:00
[2025-09-24T09:24:16.012+0000] {standard_task_runner.py:57} INFO - Started process 177 to run task
[2025-09-24T09:24:16.035+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'sales_dag', 'create_schemas_tables', 'scheduled__2025-09-23T00:00:00+00:00', '--job-id', '228', '--raw', '--subdir', 'DAGS_FOLDER/sales_dag.py', '--cfg-path', '/tmp/tmpwex2sgjy']
[2025-09-24T09:24:16.057+0000] {standard_task_runner.py:85} INFO - Job 228: Subtask create_schemas_tables
[2025-09-24T09:24:16.193+0000] {logging_mixin.py:151} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-24T09:24:16.489+0000] {task_command.py:415} INFO - Running <TaskInstance: sales_dag.create_schemas_tables scheduled__2025-09-23T00:00:00+00:00 [running]> on host 3517d925e010
[2025-09-24T09:24:17.085+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='sales_dag' AIRFLOW_CTX_TASK_ID='create_schemas_tables' AIRFLOW_CTX_EXECUTION_DATE='2025-09-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-23T00:00:00+00:00'
[2025-09-24T09:24:17.163+0000] {base.py:73} INFO - Using connection ID '***_conn' for task execution.
[2025-09-24T09:24:17.209+0000] {sql.py:418} INFO - Running statement: 
    CREATE SCHEMA IF NOT EXISTS sales;
    CREATE SCHEMA IF NOT EXISTS DW_sales;
    , parameters: None
[2025-09-24T09:24:17.226+0000] {sales_dag.py:121} INFO - Schemas created successfully
[2025-09-24T09:24:17.281+0000] {base.py:73} INFO - Using connection ID '***_conn' for task execution.
[2025-09-24T09:24:17.324+0000] {sql.py:418} INFO - Running statement: 
    CREATE TABLE IF NOT EXISTS sales.transactions_staging (
        unique_row_id text,
        InvoiceNo text,
        StockCode text,
        Description text,
        Quantity double precision,
        InvoiceDate timestamp,
        UnitPrice double precision,
        CustomerID double precision,
        Country text
    );
    CREATE TABLE IF NOT EXISTS sales.transactions (
        unique_row_id text PRIMARY KEY,
        InvoiceNo text,
        StockCode text,
        Description text,
        Quantity double precision,
        InvoiceDate timestamp,
        UnitPrice double precision,
        CustomerID double precision,
        Country text
    );
    , parameters: None
[2025-09-24T09:24:17.327+0000] {sales_dag.py:123} INFO - Tables created successfully
[2025-09-24T09:24:17.328+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-09-24T09:24:17.407+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=sales_dag, task_id=create_schemas_tables, execution_date=20250923T000000, start_date=20250924T092415, end_date=20250924T092417
[2025-09-24T09:24:17.454+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-24T09:24:17.536+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
